lib/faker/string.ex

```diff
@@ -24,4 +24,24 @@ defmodule Faker.String do
     |> Base.encode64()
     |> binary_part(0, length)
   end
+
+  @doc """
+  Returns a random hexadecimal String
+  ## Examples
+      iex> Faker.String.hexadecimal()
```

Comment:
Even though it returns a String, hexadecimal is a number according to its definition.

```suggestion
      iex> Faker.Number.hexadecimal()
```

Reply:
Here is [Faker Ruby reference](https://github.com/faker-ruby/faker/blob/master/lib/faker/default/number.rb#L122).